tests/cli/commands/test_variable_command.py

```diff
@@ -0,0 +1,132 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+import os
+import unittest
+
+from airflow import models
+from airflow.bin import cli
+from airflow.cli.commands import variable_command
+from airflow.models import Variable
+
+DEV_NULL = "/dev/null"
```

Comment:
```suggestion
DEV_NULL = os.devnull
```

(or just us that value directly in the tests)

Reply:
Fixed.